src/main/java/com/altenheim/kalender/controller/Factories/InjectorFactory.java

```diff
@@ -62,7 +62,7 @@ public void createServices()
         var systemNotificationsCt = new SystemNotificationsController(settings, calendarEntriesModel);
         allViews = new ViewRootsModel(plannerVCt, searchVCt, contactsVCt, mailVCt, settingsVCt);
         guiSetup = new GuiUpdateController(jMetroStyle, allViews);
-        initialSettingsLoader = new InitialSetupController(settings, ioCt, popupViewController, websiteCt, systemNotificationsCt);
+        initialSettingsLoader = new InitialSetupController(settings, ioCt, popupViewController, websiteCt, systemNotificationsCt, customCalendarView);
```

Comment:
warum wird die hier übergeben? Die Initialisierung kann doch direkt beim Bauen des Objektes erfolgen, wozu in der Init?

Reply:
ja hast recht, hatte nur gedacht, dass man es so besser Nachvollziehen kann, was alles gestartet wird und im Hintergrund läuft